butterknife/src/test/java/butterknife/ButterKnifeTest.java

```diff
@@ -152,4 +154,16 @@ public void resetViewsCache() {
           + " for yo mama was not found. If this view is optional add '@Nullable' annotation.");
     }
   }
+
+  @Config(shadows = EditModeShadowView.class)
+  @Test public void finderThrowsLessNiceErrorInEditMode() {
+    View view = new View(Robolectric.application);
+    try {
+      ButterKnife.Finder.VIEW.findRequiredView(view, android.R.id.button1, "yo mama");
```

Comment:
May be adding `fail()` after calling `findRequiredView` could be a possibility. Otherwise the test wont do anything if the exception is not thrown.